### DISCUSSION TITLE
新版本激活，感谢大佬

### DISCUSSION THREAD

**Discussion by zhoubei9** [LFTEC/LearningFucker#57] [General]
dqtKb2zywEEd3zYzUenEupspdfs02PAOEg25otwc70NgxFbb9ZZRnZDmEmmN3E5x7Wd
S1MGaSEZeD6Rx7OHmUkjqFX2JYzeFpMejlquV+ApWsdUUfGFKLmu+R7bEVT9wLrugpq
lmWu5Pkxi77NT1pELymez4KfoaN9TojmQNFYiW/vQhZ605A27gAAtjJOgyIVyMmJyWc
tMPE1pe7x2YIsGCbe2i/iugjnN8JyD4Q+RQHasqR7AWhGjG2qLSbff3aPF2KbPiOArw
4A8zYKf/ulODHKeNAOW2Yw482Oe2V98XnIioH57EwhY1Dtioc+vdEUD7dLe8FpmbNJd
BuN1zLVlEIXQDoeumgC6w/mLmOwx5ftabjcKtoHjfVWX09Ydp/H8PhVveVZQWwiIlPn
3RjyC1XWHa4R/cJ5IcK9/95Zw0CkbdjlGtfkZywuxjpSUtf7B2SPUrHl88iX5ADr+si
TYF4c/0c/JQA/qwtCTmMDql0nfjNlmIaLET2lDaImXYOXpfzYmNVnx+jTr2VJri37U3
xn8aKXNW4Aq3YXGBBbg//qCBd8duWd3e+t8r3xToHTKuy+PVml4skFcO6v842uPKHJD
ROuztX0qd622kAcdtaumRsFQHPwdleFXEgYkbIa3ycB2SOTlhTIgjhedGe61U3e9qMv
jNDAM5zgZFF2w3VqWUIx+BKfVdgKlGQmkSM2KVB+Bw8iAr34EMUCV9twRCMDbwDVGMF
sFTOx7p+DQfnU1EOdzm9sQVm9V5MgRzTq8OgCHGCl+QzEaCj6vNbtWJn7rPbtUCl/HI
dViOV3mJj4Li5r49Uc3lzsPofABrV0yMymypegqymgCYGoybsmwwI/GVa3qUMkH0JfO
tomKxDyut1dvrEtD9qlzoGs9un4e1MfI+QBIm2MQGBwy+M1ohi40NPbqUaOQxnnzIrR
DfZNbUnH7/Xko3kQJH/P719vwnNlQv9VAjVztpEz+D/iD79Qiit+ylXHexUuweet1UU
JBhUOrGkeEIfBr4QBmcTgh4REg3GVZb1WC28zxAbKScEpl3r8E55AFiXTWr5rW/LmjU
lYqOX4XHIhjG4QCvTRwDtFserhF10/8NyoUYv8L73dTPaK9mEdvQq48y64G3SZM5Ues
PvEK7xbFLd76wvovTWWQb/DLtfJRxy3cVpx7JbESQK7rWzVMiB3WRUP9smyXDMM9PzG
dnwjwUc2V3IiEEqJSn0x4JuZEwItNtqLNyldgBN5rB7yVdwf6MNIq4FWaGp6L5VBXDm
31ZfoShTlJQsS0F5i6ofG6dIZdjvHwgaqwbvhTZ0uNpZCsuPyls5tqZIxHj0XwN6lz7
D07xuAu02d+3/fuNWV3+uHMncg==

**Comment by LFTEC** [comment]
新版本已更新，请自己激活吧

**Comment by zxc0519** [comment]
你好，我下载的是2.2版本，为什么显示应用版本过低？
